src/arch/xtensa/boot_loader.c

```diff
@@ -26,6 +26,28 @@
 
```

Comment:
@dabekjakub I would really like to see more explanations in commit message. I know that this might be obvious for you. So, please explain what does "turn off" unused memory means? How does this helps the code

Also, please try to spit the patches with one patch per logical change. What does introducing new strcmp function have with turning of unused memory? That would should be in another patch.

Reply:
Commit message was as obvious as it could be - I will add explanation that it means to power down the memory and it is to save power/battery life. However that was implicit for me.

srcmp is a part of turning of memory - its only needed in this one place itwill not be used anywhere else in bootloader so it is not separate logical change it is just to make it more readable.